README.md

```diff
@@ -33,6 +33,7 @@ docker run -d -e GRAPHITE_HOST=<graphite host> -e MESOS_MODE=slave \
 * `MESOS_HOST` - mesos host to monitor.
 * `MESOS_PORT` - mesos port number, likely `5050` for master and `5051` for slave.
 * `MESOS_VERSION` - mesos version to enable version-specific metrics.
+* `VERBOSE_LOGGING` - enable logging.  values can be `true` or `false`.  defaults to `false`.
```

Comment:
This would be more consistent wit the rest of the options:

```
* `VERBOSE_LOGGING` - logging toggle, can be `true` or `false`, `false` by default.
```

Then squash and LGTM.